### DISCUSSION TITLE
Wireguard container not functioning after installation

### DISCUSSION THREAD

**Discussion by Kroesss** [tteck/Proxmox#3582] [Ideas]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

After installing the Wireguard-container with the helper-script (using default settings), the connection from my device could be set-up, but there was no traffic at all coming through. After help from some help from Reddit, it appears to be connected with the removal of pivpn. The Reddit-user suggested additional commands that install pivpn, and after that, everything worked out of the box.

Link to reddit-post with the solution; https://www.reddit.com/r/homelab/comments/1ezksad/comment/ljn2oz4/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

### What settings are you currently utilizing?

Default Settings

### Which Linux distribution are you employing?

Debian 12

### Please provide detailed steps to reproduce the issue.

First install the wireguard-container with all default settings, then log into WGDashboard and create a configuration with a peer. Connect phone with the server and try to get any traffic (either a website or something from another element within the local network).

**Comment by tteck** [comment]
Have you tried with today's commits?  
BTW, your solution is using PIVPN.

**Reply by Kroesss** [reply]
No, today I reinstalled it as the user on reddit suggested, and applied the fix before trying it out, as that seemed important for the fix.

I know it is using pivpn, but since it wasn't working without it (at least before today), I figured it was better than a non-functioning installation.

Is the problem I was running into fixed in today's version? Then I will happily try that, but I just wanted to make you aware of the issue I was having.

**Reply by Kroesss** [reply]
@tteck  So I just tried installing the script again, and apparently, the commit you mentioned seems to have fixed the issue. Thanks!

Like I said, I just wanted you to be aware of the issue I was facing, but apparently you already were :)
